index.html

```diff
@@ -0,0 +1,159 @@
+          <div class="article__publishDate">Monday, June 7th, 9:22pm</div>
+        </header>
+        <section>
+          <div class="article__content">
+            <p>Stands out like a troppo when she'll be right spit the dummy. bitzer piece of piss he's got a massive corker. Stands out like a rack off mate trent from punchy rollie. bingle mate he's got a massive bush telly. As dry as a brizzie to it'll be pot. He's got a massive mickey mouse mate no worries as stands out like lizard drinking. We're going freckle my grab us a franger.</p>
+
+            <p> going stonkered. dunny when she'll be right sheila. He's got a massive jillaroo mate he's got a massive hooroo. Shazza got us some top end where as dry as a aussie rules footy. <a href="/">He hasn't got a bail out how he hasn't got a cooee.</a></p>
+
+            <h2>"THIS IS A PULL QUOTE FROM THE ARTICLE."</h2>
+
+            <p>As cunning as a give it a burl piece of piss she'll be right ace!. She'll be right chokkie my as stands out like schooner. Come a cark it no dramas we're going dinky-di. <a href="/">She'll be right ace!</a> how trent from punchy strewth. As busy as a pretty spiffy also as busy as a sook. Trent from punchy good onya where she'll be right shag on a rock. Trent from punchy yabber when as cunning as a clucky. Shazza got us some bodgy bloody lets get some yobbo. Shazza got us some jug heaps lets throw a corker.</p>
+
+            <p>She'll be right his blood's worth bottling piece of mate as dry as a good oil. Trent from punchy nipper flamin lets throw a dero. You little ripper trackie dacks with as cross as a hit the turps. Come a snag how come a dinky-di. Come a pot with she'll be right fair dinkum. He hasn't got a aerial pingpong piece of piss built like a bloke. Grab us a wuss also stands out like a figjam. Mad as a cook heaps as cross as a good oil. As cunning as a dinky-di mate milk bar. Come a scratchy how trent from punchy shonky. You little ripper stubby holder piece of piss as busy as a trackies.</p>
+          </div>
+          <nav class="article__extraNav">
+            <ul>
+              <li>
+                <a href="#"><img src="./images/comment.png"></a>
+              </li>
+              <li>
+                <a href="#"><img src="./images/readlater.png"></a>
+              </li>
+              <li>
+                <a href="#"><img src="./images/share.png"></a>
+              </li>
+            </ul>
+          </nav>
+        </section>
+      </article>
+      <div class="sec">
+        <div class="sec__author">
+          <img class="sec__authorPic" src="./images/dolphin.png">
+          <div class="sec__imgOverlay"></div>
+          <div class="sec__authorName">
+            ANNIE DOLPHIN
+          </div>
+          <div class="sec__authorSocialMedia">
+            <ul>
+              <li><a href="#">Twitter</a></li>
+              <li><a href="#">Facebook</a></li>
+              <li><a href="#">Instagram</a></li>
```

Comment:
Heads up to be very careful linking to a hash location. It's not going to matter here, but html5 `pushState` frameworks/libs often respond to "hash changes", so be aware of the tech involved